bin/destroy

```diff
@@ -1,24 +1,16 @@
 #!/bin/bash
 
```

Comment:
Why not having usage?

Reply:
Same as apply - no other usage than "kn destroy" - and if this script is running, it has been called correctly